Tasks/AzurePowerShellV4/azurepowershell.ts

```diff
@@ -22,18 +26,16 @@ async function run() {
             default:
                 throw new Error(tl.loc('JS_InvalidErrorActionPreference', _vsts_input_errorActionPreference));
         }
-
         let scriptType: string = tl.getInput('ScriptType', /*required*/true);
-        let scriptPath = tl.getPathInput('ScriptPath', false);
-        let scriptInline: string = tl.getInput('Inline', false);
-        let scriptArguments: string = tl.getInput('ScriptArguments', false);
-        let _vsts_input_failOnStandardError = tl.getBoolInput('FailOnStandardError', false);
-        let targetAzurePs: string = tl.getInput('TargetAzurePs', false);
-        let customTargetAzurePs: string = tl.getInput('CustomTargetAzurePs', false);
+        let scriptPath = convertToNullIfUndefined(tl.getPathInput('ScriptPath', false));
```

Comment:
why is this conversion needed ?

Reply:
so it seems like change in behaviour for `tl.getPathInput` as its now returning `undefined` and code expects `null`

Reply:
@agrataru instead of using `convertToNullIfUndefined` why not just use following pattern:
```javascript
let scriptPath = tl.getPathInput('ScriptPath', false) || null;
```

What is the code that is expecting null?

Reply:
@amit-avit 

What is the code that is expecting null?
> answered above

Regarding `convertToNullIfUndefined` can you please let me know what is the advantage of using the pattern mentioned by you.

Reply:
It just seems that correcting code so that it works with null as well as undefined would be better than introducing `convertToNullIfUndefined`. The function you have introduced changes the parameter type to `any` which would be better if we avoid. If we move to more strict type checking it would start creating problems.

Reply:
If I am understanding your comment correctly, the code is written with null in mind and to make it work with both(`null` and `undefined`) will require lot of refactoring.

I agree with your second comment that function can be more strict in terms of type safety.

Reply:
If it involves lot of refactoring then that is out of scope of this PR but do change the implementation as above for a better type checks.

Reply:
@amit-avit done.

Reply:
Thanks, I don't see any issue from my side but I'll let others approve it including @arjgupta